config/containers/logging/configure.md

```diff
@@ -159,25 +172,26 @@ see more options.
 
 ## Limitations of logging drivers
 
-- Users of Docker Enterprise can make use of "dual logging", which enables you to use the `docker logs`
-command for any logging driver. Refer to 
-[Reading logs when using remote logging drivers](/config/containers/logging/dual-logging/) for information about 
-using `docker logs` to read container logs locally for many third party logging solutions, including: 
-
-    - syslog	
-    - gelf	
-    - fluentd	
-    - awslogs	
-    - splunk	
-    - etwlogs	
-    - gcplogs	
-    - Logentries
-    
-- When using Docker Community Engine, the `docker logs` command is only available on the following drivers:
-
+- Users of Docker Enterprise can make use of "dual logging", which enables you
+  to use the `docker logs` command for any logging driver. Refer to
+  [reading logs when using remote logging drivers](dual-logging.md) for
+  information about using `docker logs` to read container logs locally for many
+  third party logging solutions, including: 
+    - `syslog`
+    - `gelf`
+    - `fluentd`
+    - `awslogs`
+    - `splunk`
+    - `etwlogs`
+    - `gcplogs`
+    - `Logentries`
+- When using Docker Community Engine, the `docker logs` command is only available
```

Comment:
Should we update this to Docker Engine?

Reply:
This feature has been open-sourced by Mirantis (see https://github.com/moby/moby/pull/40543), so will be included in the upcoming Engine release; we'll have to rewrite some of these pages related to that, so I thought to do it as part of that

Reply:
Sure, that makes sense. Thanks.